misc/database_session_handler.sql

```diff
@@ -0,0 +1,49 @@
+CREATE DATABASE IF NOT EXISTS oiosaml;
+
+use oiosaml;
+
+DROP TABLE if EXISTS replay_tbl;
+
+CREATE TABLE replay_tbl
+(
+    assertion_id VARCHAR(255) NOT NULL,
+    access_time TIMESTAMP NOT NULL,
+    CONSTRAINT replay_assertion_id_pk PRIMARY KEY (assertion_id)
+);
+
+DROP TABLE if EXISTS assertions_tbl;
+
+CREATE TABLE assertions_tbl
+(
+    session_id VARCHAR(255) NOT NULL,
+    session_index VARCHAR(255) NOT NULL,
+    assertion_id VARCHAR(255) NOT NULL,
+    subject_name_id VARCHAR(255) NOT NULL,
+    access_time TIMESTAMP NOT NULL,
+    xml_object BLOB,
+    CONSTRAINT assertions_session_id_pk PRIMARY KEY (session_id)
+);
+
+CREATE INDEX assertions_session_index_idx ON assertions_tbl (session_index);
+
+DROP TABLE if EXISTS authn_requests_tbl;
+
+CREATE TABLE authn_requests_tbl
+(
+    session_id VARCHAR(255) NOT NULL,
+    access_time TIMESTAMP NOT NULL,
+    nsis_level VARCHAR(100) NOT NULL,
+    request_path VARCHAR(8000) NOT NULL,
+    xml_object BLOB,
```

Comment:
I would argue that we should always use UTF-8, so if we encode and decode using this encoding always we should be able to use CLOB. Any specific reasons database for changing the data type?

Reply:
It would be more readable for a dba, but since I'm writing base64 encoded data, the value would be minimal. 
But our data is character-based so it makes sense to switch to Clob.

Reply:
I have refactored the DatabaseSessionHandler to mitigate this change